kubernetes/ingress.yml

```diff
@@ -1,10 +1,18 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: ingress-app
+  name: app-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /
 spec:
-  defaultBackend:
-    service:
-      name: service-app
-      port:
-        number: 5000
\ No newline at end of file
+  rules:
+  - host: app.local
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: service-app
+            port:
+              number: 5000
```

Comment:
Do we really need these changes

Reply:
see comment above, it's just the default template for an ingress